pkg/data/data_test.go

```diff
@@ -0,0 +1,29 @@
+package data
+
+import "testing"
+
+func TestData(t *testing.T) {
```

Comment:
The naming can be more descriptive. `TestSendData` seems a more intuitive method name

Reply:
👍